src/Web3/Types.elm

```diff
@@ -218,3 +218,32 @@ type CallType
     | Async
     | Setter
     | Getter
+
+
+{-| Available ethereum denominations.
+-}
+type EthDenomination
+    = Wei --        1
+    | Kwei --       1000
```

Comment:
I mean unit prefixes like "kilo" and "milli" - To me, those are easily computable from every base unit. For example, why do I need to have a typed representation of `Kwei`, when I know that I can take my value in `Wei`, and just divide by 1000 to get the same amount in `Kwei`? Plus, we'd never be able to fully exhaust all possibilities across all base units. "Micro", "Nano", "Kilo", etc all exist for every single base.